backend/app/src/test/kotlin/io/tolgee/service/notification/NotificationServiceTest.kt

```diff
@@ -0,0 +1,90 @@
+package io.tolgee.service.notification
+
+import io.tolgee.AbstractSpringTest
+import io.tolgee.development.Base
+import io.tolgee.model.notifications.Notification
+import io.tolgee.model.notifications.NotificationChannel
+import io.tolgee.model.notifications.NotificationType
+import io.tolgee.model.notifications.NotificationTypeGroup
+import io.tolgee.testing.NotificationTestUtil
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+
+class NotificationServiceTest : AbstractSpringTest() {
+  @Autowired
+  private lateinit var notificationTestUtil: NotificationTestUtil
+
+  @Autowired
+  private lateinit var notificationService: NotificationService
+
+  @Autowired
+  private lateinit var notificationSettingsService: NotificationSettingsService
+
+  private lateinit var base: Base
+
+  @BeforeEach
+  fun setUp() {
+    notificationTestUtil.init()
+    base = dbPopulator.createBase()
+  }
+
+  @Test
+  fun inAppAndEmailNotificationAreCreated() {
```

Comment:
```suggestion
  fun `in app and email notifications are created`() {
```

Name all of the tests this way.